torch/distributed/algorithms/model_averaging/utils.py

```diff
@@ -1,13 +1,12 @@
 # flake8: noqa C101
 import itertools
-from typing import Iterator
+from typing import List, Union, Iterable, Dict
 
 import torch
 import torch.distributed as dist
 
-
 def average_parameters(
-    params: Iterator[torch.nn.Parameter], process_group: dist.ProcessGroup
+    params: List[torch.nn.Parameter], process_group: dist.ProcessGroup
```

Comment:
Since now the input is a list instead of an iterator, L21 `params_it1, params_it2 = itertools.tee(params)` is now unnecessary.

Reply:
Then also no need to import `itertools` in L2.